libs/designer/src/lib/ui/panel/templatePanel/templatePanel.tsx

```diff
@@ -154,7 +114,7 @@ export const TemplatePanel = ({ createWorkflow, onClose, showCreate, workflowId
       {currentPanelView === 'createWorkflow' ? (
         <CreateWorkflowPanel panelTabs={createWorkflowPanelTabs} />
       ) : currentPanelView === 'quickView' ? (
-        <QuickViewPanel workflowId={workflowId} showCreate={showCreate} />
+        <QuickViewPanel workflowId={workflowId as string} clearDetailsOnClose={showCreate} />
```

Comment:
`undefined as string` will be `"undefined"`, is this what we want? Otherwise, could we change it to `workflowId ?? ''`

Reply:
No, when we are in QuickViewPanel workflowId should always be present, If it is not it is a bug and we will know that, putting workflowid to empty might mask it